resources/leiningen/new/kraken_http_api/service.clj

```diff
@@ -0,0 +1,35 @@
+(ns {{name}}.service
+  (:require [io.pedestal.http :as bootstrap]
+            [io.pedestal.http.route :as route]
+            [io.pedestal.http.route.definition :refer [defroutes]]
+            [io.pedestal.interceptor :refer [interceptor]]
+            [ring.util.response :as ring-resp]
+            [turbovote.resource-config :refer [config]]
+            [pedestal-toolbox.params :refer :all]
+            [pedestal-toolbox.content-negotiation :refer :all]
+            [kehaar.core :as k]
+            [clojure.core.async :refer [chan go alt! timeout]]))
+
+(def ping
```

Comment:
I think it's pretty useful even just knowing that the HTTP server is up and running.

The thing that seems weird about doing anything with the :ok queue is that it would be a message out and back into the same component. That's perfectly valid, I suppose, but I'm not sure it tells you very much.

Or did you mean send a message to the backend *-works component's :ok queue and wait for a response? The weird thing there is that these HTTP API gateways are specifically intended *not* to be tied to 1:1 relationships with backend components. They're intended to implement useful, predictable, pragmatic HTTP APIs for us and other developers. It just so happens that all the ones we've built so far only talk to one backend component each. If that ends up being true everywhere, then maybe we can bake it in further. But it feels too early to start doing that here.

Or I suppose we could gather a list of every backend component that they talk to and send an :ok message to each of them in ping... Hmm...

There could be something useful here, I'm just not sure what it should look like at this stage. Thoughts?